### PR TITLE
Add Google OAuth configuration to README and .env.dist

### PR DESCRIPTION
**Pull request by dawntomm**
### Description
`README` does not have instruction for new developers to successfully setup the dev environment. `env.dist` also does not include configuration for Google OAuth.

### Description
Update `README` and `.env.dist` with Google OAuth configuration.

resolves https://github.com/short-d/short/issues/483

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
 1. Replace the value of `FACEBOOK_CLIENT_SECRET` in `backend/.env` file with
    `App Secret`.
 
+### Create Google OAuth Application
```
Comment (byliuyang) [inline README.md]:
```suggestion
### Create Google OAuth Client ID
```

File: README.md [hunk 2]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
...
+
+Create a new credential at
```
Comment (byliuyang) [inline README.md]:
```suggestion
Create a new client ID at
```

File: README.md [hunk 3]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
...
+   [Credentials page](https://console.developers.google.com/apis/credentials) with the following steps:
```
Comment (byliuyang) [inline README.md]:
```suggestion
   [Google API Credentials](https://console.developers.google.com/apis/credentials):
```

File: README.md [hunk 4]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
...
+
+1. Click `OAuth client ID` under `Create credentials`.
```
Comment (byliuyang) [inline README.md]:
```suggestion
1. Click on `Create Credentials` and select `OAuth client ID`.
```

File: README.md [hunk 5]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
...
+
+   ![Credentials](doc/google/credentials.jpg)
+
+1. Choose `Web application` under `Application type`.
```
Comment (byliuyang) [inline README.md]:
```suggestion
1. Select `Web application` for `Application type`.
```

File: README.md [hunk 6]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
...
+
+1. Put `http://localhost/oauth/google/sign-in/callback` under `Authorized redirect URIs`, click `Create`.
```
Comment (byliuyang) [inline README.md]:
```suggestion
1. Fill in `http://localhost/oauth/google/sign-in/callback` for `Authorized redirect URIs` and click on `Create`
```

File: README.md [hunk 7]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
...
+
+   ![OAuth](doc/google/oauth.jpg)
+   
+1. Replace the value of `GOOGLE_CLIENT_ID` in `backend/.env` file with `Client ID`.
```
Comment (byliuyang) [inline README.md]:
```suggestion
1. Replace the value of `GOOGLE_CLIENT_ID` in `backend/.env` file with `Your Client ID`.
```

File: README.md [hunk 8]
```diff
@@ -149,6 +149,25 @@ git clone https://github.com/byliuyang/short.git
...
+1. Replace the value of `GOOGLE_CLIENT_SECRET` in `backend/.env` file with
+   `Client Secret`.
```
Comment (byliuyang) [inline README.md]:
```suggestion
   `Your Client Secret`.
```

File: README.md (hunk 7 above)
Comment (dawntomm) [reply README.md]: addressed comment

**Comment by dawntomm** [comment]
addressed comment, please review when you got the chance, thank you!

**Comment by byliuyang** [comment]
Could you please include the header in the screenshot so that it's easier for reader to figure out the location of credentials box? doc/google/credentials.jpg Thanks!

**Comment by byliuyang** [comment]
> Could you please include the Google Cloud Platform header in the screenshot so that it's easier for reader to figure out the location of credentials box? doc/google/credentials.jpg Thanks!

Discussed offline. This is resolved.

**Pull request merged**